pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/actions/GutterMarker.kt

```diff
@@ -3,16 +3,16 @@
 
 package com.amos.pitmutationmate.pitmutationmate.actions
 
+import com.amos.pitmutationmate.pitmutationmate.icons.Icons
 import com.intellij.execution.lineMarker.RunLineMarkerContributor
-import com.intellij.icons.AllIcons
 import com.intellij.psi.PsiClass
 import com.intellij.psi.PsiElement
 import org.jetbrains.kotlin.psi.KtClass
 import javax.swing.Icon
 
 class GutterMarker : RunLineMarkerContributor() {
     override fun getInfo(psielement: PsiElement): Info? {
-        val gutterIcon: Icon = AllIcons.General.ArrowRight
+        val gutterIcon: Icon = Icons.RunButton
         if (psielement is PsiClass) {
```

Comment:
```suggestion
        if (psielement.context is PsiClass && psielement.text.equals("class")) {
            val toolTipProvider: (PsiElement) -> String = { _ -> "Run PIT MutationMate on '${(psielement.context as PsiClass).name}'" }
            val fqn = (psielement.context as PsiClass).qualifiedName
            val action: Array<GutterAction?> = arrayOf(fqn?.let { GutterAction(it) })
            return Info(gutterIcon, action, toolTipProvider)
        }
```

I had a look at your problem with replacing the gutter Icon. This could be one solution to it. When really helps me when working with PsiElements is [this](https://plugins.jetbrains.com/plugin/227-psiviewer) plugin. There you can check how the PsiTree is built for the file that you have currently opened.
For Kotlin it will be slightly different probably.
I hope this helps you!